.goreleaser-template.yaml

```diff
@@ -161,65 +164,65 @@ builds:
 
 report_sizes: true
 
-nfpms:
-  - vendor: OpenBao
-    homepage: https://openbao.org
-    maintainer: OpenBao <openbao@lists.lfedge.org>
-    description: |
-      OpenBao exists to provide a software solution to manage, store, and distribute
-      sensitive data including secrets, certificates, and keys.
-    license: MPL-2.0
-    formats:
-      # - apk
-      - deb
-      - rpm
-      - archlinux
-    dependencies:
-      - openssl
-    provides:
-      - bao
-    bindir: /usr/bin
-    section: default
-    contents:
-      - src: ./LICENSE
-        dst: /usr/share/doc/openbao/copyright
-        file_info:
-          mode: 0644
-      - src: ./.release/linux/package/etc/openbao/openbao.env
-        dst: /etc/openbao/openbao.env
-        file_info:
-          mode: 0644
-      - src: ./.release/linux/package/etc/openbao/openbao.hcl
-        dst: /etc/openbao/openbao.hcl
-        file_info:
-          mode: 0644
-      - src: ./.release/linux/package/usr/lib/systemd/system/openbao.service
-        dst: /usr/lib/systemd/system/openbao.service
-        file_info:
-          mode: 0644
-    scripts:
-      preinstall: ".release/linux/preinst"
-      postinstall: ".release/linux/postinst"
-      postremove: ".release/linux/postrm"
-    overrides:
-      deb:
-        dependencies:
-          - openssl
-      rpm:
-        dependencies:
-          - openssl
-      archlinux:
-        dependencies:
-          - openssl
-    deb:
-      signature:
-        key_file: "{{ .Env.GPG_KEY_FILE }}"
-    rpm:
-      signature:
-        key_file: "{{ .Env.GPG_KEY_FILE }}"
+#LINUXONLY#nfpms:
```

Comment:
You don't need to comment the `nfpms` section, it is only run for linux targets anyways

Reply:
I had to, otherwise pipelines would fail.... See [this pipeline run](https://github.com/cipherboy/openbao/actions/runs/9966641627/job/27539029287#step:10:513):

![image](https://github.com/user-attachments/assets/6a521f82-873c-47d1-9dc0-6a60af1d2c6e)

<details>

<summary> Suffix of build log </summary>

```

2024-07-17T01:30:22.8289266Z go: downloading github.com/cloudflare/circl v1.3.7
2024-07-17T01:30:23.1181581Z go: downloading github.com/go-ole/go-ole v1.2.6
2024-07-17T01:30:23.1345076Z go: downloading github.com/tklauser/numcpus v0.4.0
2024-07-17T01:30:23.1635363Z go: downloading github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932
2024-07-17T01:30:23.1689176Z go: downloading github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
2024-07-17T01:30:23.2288886Z go: downloading github.com/circonus-labs/circonusllhist v0.1.3
2024-07-17T01:30:23.2363307Z go: downloading github.com/opentracing/opentracing-go v1.2.1-0.20220228012449-10b1cf09e00b
2024-07-17T01:30:23.2463976Z go: downloading github.com/uber/jaeger-client-go v2.30.0+incompatible
2024-07-17T01:30:23.2906191Z go: downloading github.com/Azure/go-autorest/logger v0.2.1
2024-07-17T01:30:23.2955583Z go: downloading github.com/Azure/go-autorest/tracing v0.6.0
2024-07-17T01:30:23.3003018Z go: downloading github.com/Azure/go-autorest/autorest/mocks v0.4.2
2024-07-17T01:30:23.3004125Z go: downloading cloud.google.com/go/iam v1.1.5
2024-07-17T01:30:23.3005025Z go: downloading github.com/sony/gobreaker v0.5.0
2024-07-17T01:30:23.3068963Z go: downloading github.com/dnephin/pflag v1.0.7
2024-07-17T01:30:23.3119939Z go: downloading github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
2024-07-17T01:30:23.3191413Z go: downloading golang.org/x/exp/typeparams v0.0.0-20240604190554-fc45aab8b7f8
2024-07-17T01:30:23.3193574Z go: downloading golang.org/x/mod v0.18.0
2024-07-17T01:30:23.3300862Z go: downloading github.com/jcmturner/aescts/v2 v2.0.0
2024-07-17T01:30:23.3350532Z go: downloading github.com/jcmturner/rpc/v2 v2.0.3
2024-07-17T01:30:23.3428070Z go: downloading github.com/denverdino/aliyungo v0.0.0-20190125010748-a747050bb1ba
2024-07-17T01:30:23.3452176Z go: downloading github.com/Azure/go-autorest/autorest/azure/auth v0.5.12
2024-07-17T01:30:23.3453080Z go: downloading github.com/digitalocean/godo v1.7.5
2024-07-17T01:30:23.3524708Z go: downloading github.com/linode/linodego v0.7.1
2024-07-17T01:30:23.3626104Z go: downloading github.com/hashicorp/mdns v1.0.4
2024-07-17T01:30:23.3684255Z go: downloading github.com/gophercloud/gophercloud v0.1.0
2024-07-17T01:30:23.6907966Z go: downloading github.com/packethost/packngo v0.1.1-0.20180711074735-b9cb5096f54c
2024-07-17T01:30:23.6909487Z go: downloading github.com/nicolai86/scaleway-sdk v1.10.2-0.20180628010248-798f60e20bb2
2024-07-17T01:30:23.6910896Z go: downloading github.com/softlayer/softlayer-go v0.0.0-20180806151055-260589d94c7d
2024-07-17T01:30:23.7025572Z go: downloading github.com/tencentcloud/tencentcloud-sdk-go v1.0.162
2024-07-17T01:30:23.7544389Z go: downloading github.com/joyent/triton-go v1.7.1-0.20200416154420-6801d15b779f
2024-07-17T01:30:23.7935042Z go: downloading github.com/hashicorp/vic v1.5.1-0.20190403131502-bbfe86ec9443
2024-07-17T01:30:23.9693286Z go: downloading github.com/vmware/govmomi v0.18.0
2024-07-17T01:30:24.1853098Z go: downloading go.opentelemetry.io/otel/metric v1.19.0
2024-07-17T01:30:24.1854406Z go: downloading github.com/go-logr/stdr v1.2.2
2024-07-17T01:30:24.1935240Z go: downloading github.com/opencontainers/runc v1.1.12
2024-07-17T01:30:24.1936118Z go: downloading github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5
2024-07-17T01:30:24.1939977Z go: downloading github.com/moby/term v0.0.0-20221205130635-1aeaba878587
2024-07-17T01:30:24.2014038Z go: downloading github.com/go-openapi/swag v0.22.4
2024-07-17T01:30:24.2196712Z go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
2024-07-17T01:30:24.2213586Z go: downloading github.com/modern-go/reflect2 v1.0.2
2024-07-17T01:30:24.2254696Z go: downloading github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
2024-07-17T01:30:24.2300210Z go: downloading github.com/onsi/ginkgo/v2 v2.9.1
2024-07-17T01:30:24.2358994Z go: downloading github.com/googleapis/enterprise-certificate-proxy v0.3.2
2024-07-17T01:30:24.2552352Z go: downloading github.com/nxadm/tail v1.4.8
2024-07-17T01:30:24.2687908Z go: downloading github.com/tv42/httpunix v0.0.0-20191220191345-2ba4b9c3382c
2024-07-17T01:30:24.2743697Z go: downloading github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
2024-07-17T01:30:24.2859778Z go: downloading github.com/jmespath/go-jmespath v0.4.0
2024-07-17T01:30:24.2972139Z go: downloading gopkg.in/ini.v1 v1.66.2
2024-07-17T01:30:24.3305275Z go: downloading github.com/Azure/azure-sdk-for-go/sdk/internal v1.8.0
2024-07-17T01:30:24.3600040Z go: downloading github.com/Azure/azure-sdk-for-go/sdk/keyvault/internal v0.7.1
2024-07-17T01:30:24.3706550Z go: downloading github.com/AzureAD/microsoft-authentication-library-for-go v1.2.2
2024-07-17T01:30:24.4020862Z go: downloading github.com/golang-jwt/jwt/v5 v5.2.1
2024-07-17T01:30:24.4021934Z go: downloading github.com/Azure/go-autorest/autorest/adal v0.9.22
2024-07-17T01:30:24.4024781Z go: downloading github.com/uber/jaeger-lib v2.4.1+incompatible
2024-07-17T01:30:24.4119910Z go: downloading github.com/fsnotify/fsnotify v1.6.0
2024-07-17T01:30:24.4276063Z go: downloading github.com/Azure/go-autorest/autorest/validation v0.3.1
2024-07-17T01:30:24.4338238Z go: downloading github.com/Azure/go-autorest/autorest/azure/cli v0.4.5
2024-07-17T01:30:24.4341527Z go: downloading github.com/dimchansky/utfbom v1.1.1
2024-07-17T01:30:24.4342691Z go: downloading github.com/google/go-querystring v1.1.0
2024-07-17T01:30:24.4392537Z go: downloading github.com/miekg/dns v1.1.43
2024-07-17T01:30:24.4406532Z go: downloading gopkg.in/resty.v1 v1.12.0
2024-07-17T01:30:24.4414446Z go: downloading github.com/dnaeon/go-vcr v1.1.0
2024-07-17T01:30:24.4636987Z go: downloading github.com/renier/xmlrpc v0.0.0-20170708154548-ce4a1a486c03
2024-07-17T01:30:24.4741133Z go: downloading github.com/jarcoal/httpmock v1.2.0
2024-07-17T01:30:24.4755929Z go: downloading github.com/abdullin/seq v0.0.0-20160510034733-d5467c17e7af
2024-07-17T01:30:24.4906363Z go: downloading github.com/containerd/continuity v0.4.2
2024-07-17T01:30:24.4942121Z go: downloading github.com/xeipuuv/gojsonschema v1.2.0
2024-07-17T01:30:24.5259289Z go: downloading github.com/mailru/easyjson v0.7.7
2024-07-17T01:30:24.5444902Z go: downloading github.com/go-openapi/jsonreference v0.20.2
2024-07-17T01:30:24.5518810Z go: downloading github.com/creack/pty v1.1.18
2024-07-17T01:30:24.5521659Z go: downloading github.com/emicklei/go-restful/v3 v3.10.1
2024-07-17T01:30:24.5553151Z go: downloading github.com/morikuni/aec v1.0.0
2024-07-17T01:30:24.5724379Z go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
2024-07-17T01:30:24.5741872Z go: downloading github.com/jmespath/go-jmespath/internal/testify v1.5.1
2024-07-17T01:30:24.5769025Z go: downloading github.com/pkg/browser v0.0.0-20240102092130-5ac0b6a4141c
2024-07-17T01:30:24.5798191Z go: downloading github.com/Azure/go-autorest/autorest/date v0.3.0
2024-07-17T01:30:24.5853254Z go: downloading github.com/sean-/seed v0.0.0-20170313163322-e2103e2c3529
2024-07-17T01:30:24.5864382Z go: downloading github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415
2024-07-17T01:30:24.5904994Z go: downloading github.com/josharian/intern v1.0.0
2024-07-17T01:30:24.5921633Z go: downloading github.com/Azure/go-ansiterm v0.0.0-20230124172434-306776ec8161
2024-07-17T01:30:24.6123976Z go: downloading github.com/go-openapi/jsonpointer v0.20.0
2024-07-17T01:30:24.6164540Z go: downloading github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb
2024-07-17T01:30:24.6166273Z go: downloading github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
2024-07-17T01:30:24.6194395Z go: downloading github.com/google/pprof v0.0.0-20210720184732-4bb14d4b1be1
2024-07-17T01:30:24.8018554Z [1;94m    •[0m running                                        [1;94mhook[0m=go generate ./...
2024-07-17T01:30:24.8020465Z [1;97m    •[0m running                                        [1;97mcmd[0m=[go generate ./...] [1;97mdir[0m=
2024-07-17T01:30:25.1817762Z [1;94m    •[0m running                                        [1;94mhook[0m=make bootstrap
2024-07-17T01:30:25.1818887Z [1;97m    •[0m running                                        [1;97mcmd[0m=[make bootstrap] [1;97mdir[0m=
2024-07-17T01:30:25.1845762Z go generate -tags tools tools/tools.go
2024-07-17T01:30:50.1471516Z [1;94m    •[0m [3;2mtook: 49s[0m
2024-07-17T01:30:50.1472466Z [1;94m  •[0m [1mchecking distribution directory[0m
2024-07-17T01:30:50.1473496Z [1;97m    •[0m dist doesn't exist, creating empty directory
2024-07-17T01:30:50.1478759Z [1;94m  •[0m [1msetting up metadata[0m
2024-07-17T01:30:50.1480001Z [1;94m  •[0m [1mstoring release metadata[0m
2024-07-17T01:30:50.1481646Z [1;94m    •[0m writing                                        [1;94mfile[0m=dist/metadata.json
2024-07-17T01:30:50.1483377Z [1;97m    •[0m added new artifact                             [1;97mname[0m=metadata.json [1;97mtype[0m=Metadata [1;97mpath[0m=dist/metadata.json
2024-07-17T01:30:50.1485425Z [1;94m  •[0m [1mloading go mod information[0m
2024-07-17T01:30:50.1528172Z [1;94m  •[0m [1mbuild prerequisites[0m
2024-07-17T01:30:50.1529212Z [1;97m  •[0m skipped checking go.mod
2024-07-17T01:30:50.1529921Z [1;97m  •[0m skipped proxying go module
2024-07-17T01:30:50.1530687Z [1;94m  •[0m [1mwriting effective config file[0m
2024-07-17T01:30:50.1536936Z [1;94m    •[0m writing                                        [1;94mconfig[0m=dist/config.yaml
2024-07-17T01:30:50.1537919Z [1;94m  •[0m [1mbuilding binaries[0m
2024-07-17T01:30:50.1542484Z [1;97m    •[0m building                                       [1;97mbuild[0m={builds-linux [illumos] [amd64 arm arm64 ppc64le riscv64 s390x] [6] [hardfloat] [v1] [] [{darwin    } {dragonfly    } {freebsd    } {illumos    } {netbsd    } {openbsd    } {solaris    } {wasip1    } {windows    }] . . bao {[] []} go {{ .CommitTimestamp }} false go build false false   { [-X github.com/openbao/openbao/version.fullVersion={{.Version}} -X github.com/openbao/openbao/version.GitCommit={{.Commit}} -X github.com/openbao/openbao/version.BuildDate={{ .Date }}] [] [] [] [] [CGO_ENABLED=0]} []}
2024-07-17T01:30:50.1551935Z [1;97m    •[0m building                                       [1;97mbuild[0m={builds-other [illumos] [amd64 arm arm64 riscv64] [6 7] [hardfloat] [v1] [illumos_amd64_v1] [{linux    } {darwin  6  } {darwin arm   } {darwin ppc64le   } {darwin riscv64   } {darwin s390x   } {darwin wasm   } {dragonfly arm   } {dragonfly arm64   } {dragonfly riscv64   } {dragonfly wasm   } {freebsd  7  } {freebsd ppc64le   } {freebsd s390x   } {freebsd wasm   } {illumos arm   } {illumos arm64   } {illumos ppc64le   } {illumos s390x   } {illumos wasm   } {netbsd ppc64le   } {netbsd riscv64   } {netbsd s390x   } {netbsd wasm   } {netbsd  7  } {openbsd ppc64le   } {openbsd riscv64   } {openbsd s390x   } {openbsd wasm   } {openbsd  7  } {solaris arm   } {solaris arm64   } {solaris ppc64le   } {solaris s390x   } {solaris wasm   } {wasip1 amd64   } {wasip1 arm   } {wasip1 arm64   } {wasip1 riscv64   } {windows arm 7  } {windows ppc64le   } {windows riscv64   } {windows s390x   } {windows wasm   }] . . bao {[] []} go {{ .CommitTimestamp }} false go build false false   { [-X github.com/openbao/openbao/version.fullVersion={{.Version}} -X github.com/openbao/openbao/version.GitCommit={{.Commit}} -X github.com/openbao/openbao/version.BuildDate={{ .Date }}] [] [] [] [] [CGO_ENABLED=0]} []}
2024-07-17T01:30:50.1557884Z [1;94m    •[0m building                                       [1;94mbinary[0m=dist/builds-other_illumos_amd64_v1/bao
2024-07-17T01:30:50.1558895Z [1;97m    •[0m env "CGO_ENABLED=0" evaluated to "CGO_ENABLED=0"
2024-07-17T01:30:50.1559438Z [1;97m    •[0m running
2024-07-17T01:32:40.3584707Z [1;97m    •[0m added new artifact                             [1;97mname[0m=bao [1;97mtype[0m=Binary [1;97mpath[0m=dist/builds-other_illumos_amd64_v1/bao
2024-07-17T01:32:40.3586141Z [1;94m    •[0m [3;2mtook: 1m50s[0m
2024-07-17T01:32:40.3586728Z [1;97m  •[0m skipped universal binaries
2024-07-17T01:32:40.3587551Z [1;97m  •[0m skipped sign & notarize macOS binaries
2024-07-17T01:32:40.3588114Z [1;97m  •[0m skipped upx
2024-07-17T01:32:40.3588608Z [1;94m  •[0m [1mgenerating changelog[0m
2024-07-17T01:32:40.3589216Z [1;97m    •[0m token type                                     [1;97mtype[0m=github
2024-07-17T01:32:40.3604961Z [1;97m    •[0m git command result                             [1;97margs[0m=[-c log.showSignature=false rev-parse --is-inside-work-tree] [1;97mstdout[0m=true [1;97mstderr[0m=
2024-07-17T01:32:40.3620605Z [1;97m    •[0m git command result                             [1;97margs[0m=[-c log.showSignature=false ls-remote --get-url] [1;97mstdout[0m=https://github.com/cipherboy/openbao [1;97mstderr[0m=
2024-07-17T01:32:40.3622659Z [1;97m    •[0m got git url                                    [1;97mrawurl[0m=https://github.com/cipherboy/openbao
2024-07-17T01:32:40.3624089Z [1;97m    •[0m parsed url                                     [1;97mowner[0m=cipherboy [1;97mname[0m=openbao
2024-07-17T01:32:40.8479278Z [1;97m    •[0m not grouping entries
2024-07-17T01:32:40.8480136Z [1;94m    •[0m writing                                        [1;94mchangelog[0m=dist/CHANGELOG.md
2024-07-17T01:32:40.8481109Z [1;94m  •[0m [1marchives[0m
2024-07-17T01:32:40.8481827Z [1;97m    •[0m group illumosamd64v1 has 1 binaries
2024-07-17T01:32:40.8482710Z [1;94m    •[0m creating                                       [1;94marchive[0m=dist/bao_2.0.0-testing2_Illumos_x86_64.tar.gz
2024-07-17T01:32:40.9516371Z [1;97m    •[0m adding file: CHANGELOG.md as CHANGELOG.md
2024-07-17T01:32:40.9517817Z [1;97m    •[0m adding file: LICENSE as LICENSE
2024-07-17T01:32:40.9522308Z [1;97m    •[0m adding file: README.md as README.md
2024-07-17T01:32:40.9528914Z [1;97m    •[0m adding file: changelog/120.txt as changelog/120.txt
2024-07-17T01:32:40.9533412Z [1;97m    •[0m adding file: changelog/170.txt as changelog/170.txt
2024-07-17T01:32:40.9534606Z [1;97m    •[0m adding file: changelog/173.txt as changelog/173.txt
2024-07-17T01:32:40.9535634Z [1;97m    •[0m adding file: changelog/181.txt as changelog/181.txt
2024-07-17T01:32:40.9538477Z [1;97m    •[0m adding file: changelog/184.txt as changelog/184.txt
2024-07-17T01:32:40.9539524Z [1;97m    •[0m adding file: changelog/201.txt as changelog/201.txt
2024-07-17T01:32:40.9540466Z [1;97m    •[0m adding file: changelog/202.txt as changelog/202.txt
2024-07-17T01:32:40.9541467Z [1;97m    •[0m adding file: changelog/209.txt as changelog/209.txt
2024-07-17T01:32:40.9542454Z [1;97m    •[0m adding file: changelog/210.txt as changelog/210.txt
2024-07-17T01:32:40.9543448Z [1;97m    •[0m adding file: changelog/211.txt as changelog/211.txt
2024-07-17T01:32:40.9544453Z [1;97m    •[0m adding file: changelog/212.txt as changelog/212.txt
2024-07-17T01:32:40.9545433Z [1;97m    •[0m adding file: changelog/215.txt as changelog/215.txt
2024-07-17T01:32:40.9546411Z [1;97m    •[0m adding file: changelog/221.txt as changelog/221.txt
2024-07-17T01:32:40.9547061Z [1;97m    •[0m adding file: changelog/224.txt as changelog/224.txt
2024-07-17T01:32:40.9548037Z [1;97m    •[0m adding file: changelog/238.txt as changelog/238.txt
2024-07-17T01:32:40.9548999Z [1;97m    •[0m adding file: changelog/269.txt as changelog/269.txt
2024-07-17T01:32:40.9549950Z [1;97m    •[0m adding file: changelog/275.txt as changelog/275.txt
2024-07-17T01:32:40.9551209Z [1;97m    •[0m adding file: changelog/294.txt as changelog/294.txt
2024-07-17T01:32:40.9552281Z [1;97m    •[0m adding file: changelog/298.txt as changelog/298.txt
2024-07-17T01:32:40.9553247Z [1;97m    •[0m adding file: changelog/313.txt as changelog/313.txt
2024-07-17T01:32:40.9554172Z [1;97m    •[0m adding file: changelog/321.txt as changelog/321.txt
2024-07-17T01:32:40.9555670Z [1;97m    •[0m adding file: changelog/36.txt as changelog/36.txt
2024-07-17T01:32:40.9556672Z [1;97m    •[0m adding file: changelog/363.txt as changelog/363.txt
2024-07-17T01:32:40.9557673Z [1;97m    •[0m adding file: changelog/365.txt as changelog/365.txt
2024-07-17T01:32:40.9558645Z [1;97m    •[0m adding file: changelog/401.txt as changelog/401.txt
2024-07-17T01:32:40.9559656Z [1;97m    •[0m adding file: changelog/50.txt as changelog/50.txt
2024-07-17T01:32:40.9560563Z [1;97m    •[0m adding file: changelog/76.txt as changelog/76.txt
2024-07-17T01:32:40.9561330Z [1;97m    •[0m adding file: changelog/85.txt as changelog/85.txt
2024-07-17T01:32:40.9562359Z [1;97m    •[0m adding file: changelog/README.md as changelog/README.md
2024-07-17T01:32:40.9563091Z [1;97m    •[0m adding file: changelog/changelog.tmpl as changelog/changelog.tmpl
2024-07-17T01:32:40.9565111Z [1;97m    •[0m adding file: changelog/note.tmpl as changelog/note.tmpl
2024-07-17T01:32:40.9566311Z [1;97m    •[0m adding file: dist/builds-other_illumos_amd64_v1/bao as bao
2024-07-17T01:32:52.9735283Z [1;97m    •[0m added new artifact                             [1;97mname[0m=bao_2.0.0-testing2_Illumos_x86_64.tar.gz [1;97mtype[0m=Archive [1;97mpath[0m=dist/bao_2.0.0-testing2_Illumos_x86_64.tar.gz
2024-07-17T01:32:52.9744314Z [1;94m    •[0m [3;2mtook: 12s[0m
2024-07-17T01:32:52.9745057Z [1;97m  •[0m skipped creating source archive
2024-07-17T01:32:52.9745790Z [1;94m  •[0m [1mlinux packages[0m
2024-07-17T01:32:52.9756542Z [1;91m  ⨯[0m [1mrelease failed after 2m52s[0m               [1;91merror[0m=no linux binaries found for builds []
2024-07-17T01:32:52.9783945Z ##[error]The process '/opt/hostedtoolcache/goreleaser-action/1.26.2/x64/goreleaser' failed with exit code 1
2024-07-17T01:32:52.9940363Z Post job cleanup.
2024-07-17T01:32:53.2415964Z ##[group]Removing builder
2024-07-17T01:32:53.3785103Z [command]/usr/bin/docker buildx rm builder-4866f588-b083-4fde-a30f-b317322c970a
2024-07-17T01:32:54.3104440Z builder-4866f588-b083-4fde-a30f-b317322c970a removed
2024-07-17T01:32:54.3139753Z ##[endgroup]
2024-07-17T01:32:54.3141658Z ##[group]Cleaning up certificates
2024-07-17T01:32:54.3148207Z ##[endgroup]
2024-07-17T01:32:54.3148968Z ##[group]Post cache
2024-07-17T01:32:54.3151249Z State not set
2024-07-17T01:32:54.3152012Z ##[endgroup]
2024-07-17T01:32:54.3300996Z Post job cleanup.
2024-07-17T01:32:54.4274303Z [command]/usr/bin/git version
2024-07-17T01:32:54.4317711Z git version 2.45.2
2024-07-17T01:32:54.4365593Z Temporarily overriding HOME='/home/runner/work/_temp/21f16a77-eada-48fb-8fe3-246c3f2465d6' before making global git config changes
2024-07-17T01:32:54.4366700Z Adding repository directory to the temporary git global config as a safe directory
2024-07-17T01:32:54.4379718Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/openbao/openbao
2024-07-17T01:32:54.4418860Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2024-07-17T01:32:54.4453209Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2024-07-17T01:32:54.4712013Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2024-07-17T01:32:54.4737326Z http.https://github.com/.extraheader
2024-07-17T01:32:54.4751727Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2024-07-17T01:32:54.4786780Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2024-07-17T01:32:54.5277620Z Cleaning up orphan processes
```

</details>

Tag: https://github.com/cipherboy/openbao/tree/v2.0.0-testing2 

This is because it seems to always be run, even if there are no Linux artifacts. I think this is because `build-linux` still exists as a target, even though no Linux artifacts were produced from it.

Reply:
Interesting, then the docs are misleading there

Reply:
Yeah, I will say that the run was with an earlier version, maybe the docs are for the latest version and we can remove this... but given it is tested on my branch and failed there and the pipeline should work as submitted... Let's play with it after the release? :-)